src/zepben/evolve/model/busbranch/bus_branch.py

```diff
@@ -913,9 +913,6 @@ def _process_terminal(
     has_negligible_impedance: Callable[[ConductingEquipment], bool]
 ):
     async def add_to_group(t: Terminal, _):
-        if t in tg.terminals():
-            return
-
```

Comment:
This change, along with the other, speeds up bus branch creation by ~300-400%. It is much faster to simply attempt a set insertion than to check every time.